core/domain/stats_jobs_one_off.py

```diff
@@ -1583,12 +1584,36 @@ def reduce(key, values):
             yield (key, values)
 
 
-class ResetExplorationIssuesOneOffJob(jobs.BaseMapReduceOneOffJobManager):
-    """A one-off job to reset all ExplorationIssues to a valid/empty state."""
+class WipeExplorationIssuesOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """A one-off job to wipe all `ExplorationIssuesModel`s and make them empty.
+    Any playthroughs referenced by the models are deleted as well to prevent
+    memory leaks.
+
+    This job addresses a bug discovered in the November 2020 release. The bug
+    made it to production, and corrupted a non-trivial amount of models.
+    For details, see: https://github.com/oppia/oppia/pull/11320.
+
+    Instead of recovering the corrupted models, we've decided to simply wipe
+    them all out because:
+    -   `PlaythroughModel`/`ExplorationIssuesModel` instances are not being read
+        by the frontend yet, they are only being written.
+        -   They will be read once the Improvements tab has launched, but that
+            project has been delayed by the Cloud NDB migration.
+    -   `PlaythroughModel`s current (and only) schema has bugs.
+        -   This gives us an opportunity to fix it without going through a
+            schema migration process.
+    -   `PlaythroughModel`s have new privacy requirements to meet, and there are
+        old models that do not satisfy them.
+        -   Playthroughs do not hold enough data to identify privacy-adherent
+            models without including false-positives.
+    -   Non-trivial amount of effort required to identify corrupted models due
+        to lack of debug information.
```

Comment:
This is an **excellent** background context. Thanks for adding this!

Reply:
Glad to hear :)